README.md

```diff
@@ -22,3 +22,17 @@ The following assumes you have python installed locally.
 * Install dependencies with `pip install -r requirements.txt`
 * `mkdocs serve`
 * browse to http://localhost:4000
+
```

Comment:
Can you add that you need to have the latest mkdocs-gov-au-theme, I had a older version and the tests failed.